src/main/java/com/podkor/ukrdashboard/service/feed/DataFeedService.java

```diff
@@ -0,0 +1,74 @@
+package com.podkor.ukrdashboard.service.feed;
+
+import com.podkor.ukrdashboard.dto.DataFeed;
+import com.podkor.ukrdashboard.repository.DataFeedRepository;
+import com.podkor.ukrdashboard.repository.DataTabRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.select.Elements;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.io.IOException;
+import java.util.List;
+
+import static java.util.Objects.isNull;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class DataFeedService {
+
+    protected static final String DIV_TAG = "<div class=\"tabDiv\" %s>";
+    protected static final String DIV_CLOSE_TAG = "</div>";
+
+    private final DataFeedRepository dataFeedRepository;
+    private final DataTabRepository dataTabRepository;
+
+    @Transactional(propagation = Propagation.REQUIRES_NEW)
+    @Scheduled(
+        // With constant DB it should be changed to cron
+        initialDelayString = "${config.scheduler.update-uefa-coefficients-feed.initialDelay}",
+        fixedDelayString = "${config.scheduler.update-uefa-coefficients-feed.fixedDelay}")
+    public void updateFeed() {
+        List<DataFeed> dataFeedList = dataFeedRepository.findAll();
+        dataFeedList.forEach(
+            f -> updateDataTabHtmlData(f)
+        );
+    }
+
+    public void updateDataTabHtmlData(DataFeed dataFeed) {
+        Elements elements = getRequiredElements(dataFeed);
+        String htmlData = formatHtmlData(elements.outerHtml(), dataFeed.getDataTab().getStyles())
+            .replace("/i/flag/s/", "https://terrikon.com/i/flag/s/");
```

Comment:
Is it possible to create a file with a collection of the links? To keep all of them in one place? And use constants in the code? With this poach it will be much easier updates links in future